calico-cloud/network-policy/policy-tiers/allow-tigera.mdx

```diff
@@ -1,9 +1,14 @@
 ---
-description: Understand the purpose of the allow-tigera tier and how to change its behavior.
+description: Understand how to change the behavior of the allow-tigera tier.
 ---
 
 # Change allow-tigera tier behavior
 
+:::warning
```

Comment:
Small fiddly thing, but best to have a space between the admonition fence and the content.

Reply:
Fixed.